src/Microsoft.Azure.SignalR.Management/Configuration/RetryOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.SignalR.Management;
+
+#nullable enable
+
+public class RetryOptions
```

Comment:
Give the class and `RetryMode` a more specific naming, e.g. `HttpClientRetryOptions`?